app/src/main/java/org/p2p/wallet/swap/jupiter/interactor/model/SwapTokenModel.kt

```diff
@@ -14,6 +16,8 @@ sealed interface SwapTokenModel {
     val tokenSymbol: String
     val iconUrl: String?
 
+    fun isStable(): Boolean = tokenSymbol == USDC_SYMBOL || tokenSymbol == USDT_SYMBOL
```

Comment:
maybe isStableCoin? 
we can see model.isStable() and don't understand without looking inside

Reply:
ok